src/protocols/ethereum/libs/EthplorerService.ts

```diff
@@ -37,8 +37,13 @@ export class EthplorerService {
     };
   }
 
-  async fetchTopTokens(): Promise<IToken[]> {
-    const tokens = (await this.fetchFromApi('/getTopTokens'))?.tokens || [];
+  async fetchTopTokens(): Promise<IToken[] | null> {
+    const res = await this.fetchFromApi('/getTopTokens');
+    if (res?.error?.code === 429) {
```

Comment:
I suggest to create file scoped const for the `429` error code with some comment about it.